argus/recipes/cloud/windows.py

```diff
@@ -438,3 +438,18 @@ def pre_sysprep(self):
                   "cloudbaseinit.plugins.windows.winrmlistener."
                   "ConfigWinRMListenerPlugin",
             execute_function=self._execute)
+
+
+class CloudbaseinitLocalScriptsRecipe(CloudbaseinitRecipe):
+    """Recipe for testing local scripts return codes."""
+
+    def pre_sysprep(self):
+        super(CloudbaseinitLocalScriptsRecipe, self).pre_sysprep()
+        LOG.info("Download reboot-required local script.")
+
+        cbdir = introspection.get_cbinit_dir(self._execute)
+        cmd = ("powershell Invoke-WebRequest -uri "
```

Comment:
Write the message separately and format it separately:

  msg = "..."
  msg = msg.format(...)

Travis fails because pylint can't understand this type of formatting.